mysql-test/suite/sys_vars/r/buffered_error_log_filename_basic.result

```diff
@@ -0,0 +1,20 @@
+SET @start_value = @@global.buffered_error_log_filename;
+SELECT @start_value;
+@start_value
+
+SET @@global.buffered_error_log_filename = "/tmp/bar.txt";
+SELECT @@global.buffered_error_log_filename;
+@@global.buffered_error_log_filename
+/tmp/bar.txt
+SET @@global.buffered_error_log_filename = '';
```

Comment:
when the buffered file name is empty, what is the filename chosen?

Reply:
Nothing, then it's off.

Reply:
ok, makes sense. Can you show the output of filename instead of error_log_size?

Reply:
sorry, I didn't see that you are showing filename.